search/conf.py

```diff
@@ -0,0 +1,29 @@
+"""search application configuration"""
+from appconf import AppConf
+from django.conf import settings
+
+
+class SearchAppConf(AppConf):
+    """Configuration class for a configurable CAS authentication system"""
+
+    # ENABLED = False
```

Comment:
I think you probably want to call this HAYSTACK_URL, that way you can still set it with get_var in settings.py via the `SEARCH_HAYSTACK_URL` setting (that prefix gets added by AppConf). You can then just use its value in the connection dictionary below